src/Modules/OrchardCore.Commerce/Views/Payment/PaymentConfirmationMiddleware.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿<div class="row pt-md-5">
+    <div class="col-6 mx-auto">
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                <h4 class="text-success text-center">@T["Confirming payment. Reloading in 2 seconds..."]</h4>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script>
+    const sleep = async (milliseconds) => {
+        await new Promise(resolve => {
+            return setTimeout(resolve, milliseconds)
+        });
+    };
+    const reload = async () => {
+        await sleep(2000);
+        location.reload();
+    }
+
+    reload();
```

Comment:
Do you really need a whole Promise wrapper in an inline script?
Whats wrong with just `setTimeout(() => location.reload(), 2000)`?